CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ include(Testing.cmake)
 
 find_package(Boost   REQUIRED COMPONENTS filesystem log program_options regex system thread)
 find_package(Java    REQUIRED)
-find_package(Odb     REQUIRED)
+find_package(ODB     REQUIRED)
```

Comment:
Why the change? The file is also named `FindOdb.cmake` in the root directory of the project, so it should be at least consistent in the casing.